src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.scss

```diff
@@ -14,7 +14,7 @@ table.responsive td {
   word-wrap: break-word;
 }
 
-@media screen and (max-width: 600px) {
```

Comment:
Can we use break points specified by the design system here? https://design.va.gov/design/breakpoints

Reply:
With responsive tables, breakpoints should be customized based on content.

If the effort is to create a reusable responsive table component for Formation, Is it possible to set customizable properties for the component? For example, there may be scenarios where the content shouldn't stack until a significantly smaller breakpoint, such as a table with less wide columns that can remain displayed until 320px.

Also, if the aim is to create a Formation component for a responsive table (which may be outside the scope of this work), is it possible to provide a property to add expand/collapse at smaller viewports ?

Reply:
@micahchiang I changed this to be the breakpoint from the design system. @jenstrickland This component is not the one that will be directly used in the design system, that will need to be it's own epic with its own research, its own tickets and its own PRs.

Reply:
Once this is in staging, let's take a look. I'm finding some strange behavior in other responsive implementations, so am not certain if what's in the design system will be the right choice here. Hopefully it will be, though! :)